docs/source/tutorial.rst

```diff
@@ -771,12 +771,12 @@ Python just aren't that appealing.
 
 Trio doesn't make your code run on multiple cores; in fact, as we saw
 above, it's baked into Trio's design that you never have two tasks
-running at the same time. We're not so much overcoming the GIL as
+running simultaneously. We're not so much overcoming the GIL as
```

Comment:
I believe "at the same time" was a better choice of words. It means that at any given *point* of time, only one task is running. That being said, many tasks can still make progress within a given *period* of time. The words you can search for to understand the distinction are "concurrency vs. parallelism". Hope that helps!

Reply:
Uhhhh, but "at the same time" does not work well with "simultaneously" below (see comments above), which was the whole point of the PR, to avoid this ambiguous wording.

Reply:
Sure, but now that the we use "concurrently" below, maybe that's fine?

Ultimately it's a subtle distinction that's hard to communicate to an audience that's not already aware of it :/ Maybe we should use "in parallel/concurrently"? I don't know.

In any case, this choice of words *is* better, but I won't merge this myself as I'm not a native English speaker.

Reply:
See what you mean! Tough :)